src/test/java/com/example/final_task/service/SwimmersServiceImplTest.java

```diff
@@ -2,28 +2,51 @@
 
 import com.example.final_task.entity.Swimmer;
 import com.example.final_task.exception.ResourceNotFoundException;
+import com.example.final_task.form.CreateSwimmer;
 import com.example.final_task.mapper.SwimmersMapper;
+import org.hamcrest.MatcherAssert;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.validation.BeanPropertyBindingResult;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
 
 import java.util.List;
 import java.util.Optional;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doNothing;
 import static org.mockito.Mockito.doReturn;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 
+
 @ExtendWith(MockitoExtension.class)
+@SpringBootTest
+@AutoConfigureMockMvc
 public class SwimmersServiceImplTest {
     @InjectMocks
     SwimmersServiceImpl swimmersServiceImpl;
     @Mock
     SwimmersMapper swimmersMapper;
+    @Mock
+    Validator validator;
+    @Autowired
+    MockMvc mockMvc;
+    CreateSwimmer createSwimmer = new CreateSwimmer("u1Nk","H4yjt");
+    BindingResult bindingResult;

```

Comment:
コードのスタイルはPJごと違いますがフィールド間は1行あけてほしいです。
```suggestion
    @InjectMocks
    SwimmersServiceImpl swimmersServiceImpl;
    
    @Mock
    SwimmersMapper swimmersMapper;
    
    @Mock
    Validator validator;
    
    @Autowired
    MockMvc mockMvc;
    
    CreateSwimmer createSwimmer = new CreateSwimmer("u1Nk","H4yjt");
    
    BindingResult bindingResult;

```

Reply:
6a065b1a37fc86a716b5f2e3fdd099a3b7703cfd
修正しました！
新しくクラスを作ったので結構もともとのコードが消えました！